qlib/__init__.py

```
@@ -12,6 +12,7 @@
 import platform
 import subprocess
 from .log import get_module_logger
+from .utils.serial import Serializable
```

Ruff (0.11.9)

`.utils.serial.Serializable` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)